frontend/packages/vkt/src/components/clerkExamEvent/overview/ClerkExamEventDetails.tsx

```diff
@@ -93,6 +95,7 @@ export const ClerkExamEventDetails = () => {
 
   const resetToInitialState = useCallback(() => {
     dispatch(resetClerkExamEventDetailsUpdate());
+    dispatch(resetClerkEnrollmentStatusChange());
```

Comment:
Tän rivin vois varmaan siirtää riville 131 `resetToInitialState();` tilalle. Tällöin muokkaustila ei sulkeudu jos tekee tilansiirtoja.